src/test/suite/logger.test.ts

```diff
@@ -0,0 +1,37 @@
+// import * as assert from 'assert';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as tmp from 'tmp';
+import * as assert from 'assert';
+import * as nisabaLogger from '../../logger';
+
+const nisabaOutputChannel = vscode.window.createOutputChannel("Nisaba");
```

Comment:
Very minor: is it worth having this as a separate function (e.g. `createOutputChannel`) so we can use the actual variable from the `extension.ts` without calling all of `activate`?

Reply:
This variable lives in this module, so there is no need to go through `activate`.  However we can't really do much with an [`OutputChannel`](https://code.visualstudio.com/api/references/vscode-api#OutputChannel), in particular we can't access its content, which would be useful for testing.